index.bs

```diff
@@ -347,6 +347,10 @@ The header's ABNF is:
   Sec-CH-UA-Mobile = sh-boolean
 ```
 
+Like the `Sec-CH-UA` hint above ([[#sec-ch-ua]]), since it's included in the <a
+href="https://wicg.github.io/client-hints-infrastructure/#low-entropy-table">low-entropy table</a>,
+the `Sec-CH-UA-Mobile` header will also be sent with all requests.
```

Comment:
s/sent with all requests/sent by default/?
Feature/Permissions Policy can still make sure it's not sent.

Also, should this be a note?

Reply:
That'd probably be good. There's a similar section in Sec-CH-UA, should I make that a note as well?

Reply:
Yeah, assuming it doesn't define anything

Reply:
Done

PTAL

Reply:
Done, PTAL